src/main/java/ua/price/bdd_solution/bdd_pages/pages/ItemPage.java

```diff
@@ -0,0 +1,62 @@
+package ua.price.bdd_solution.bdd_pages.pages;
+
+import net.serenitybdd.core.annotations.findby.FindBy;
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.WebDriver;
+import ua.price.bdd_solution.bdd_pages.base_page.BasePage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ItemPage extends BasePage {
+    @FindBy(xpath = "(.//a[contains(@class,'ga_mdl_tab_price')])[2]")
+    private WebElementFacade priceOffersTab;
+
+    @FindBy(xpath = ".//div[@class='sort-buttons']")
+    private WebElementFacade sortingList;
+
+    @FindBy(xpath = ".//a[@data-wa-event-label='From_cheap_to_expensive']")
+    private WebElementFacade ascendingPriceSorting;
+
+    @FindBy(xpath = ".//div[contains(@id,'pricelineAnchor')]")
+    private List<WebElementFacade> priceOffers;
+
+    @FindBy(xpath = ".//div[contains(@id,'pricelineAnchor')]//span[@class='price']")
+    private List<WebElementFacade> pricesFromPriceOffers;
+
+    @FindBy(xpath = ".//div[@class='b24-widget-button-popup b24-widget-button-popup-show']")
```

Comment:
Can we avoid using two classes at the same time?

Reply:
Done